tests/openfl/component/assigner/test_custom_assigner.py

```diff
@@ -0,0 +1,98 @@
+# Copyright (C) 2020-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+"""CustomAssigner tests."""
+
+from unittest import mock
+
+import pytest
+
+from openfl.component.aggregation_functions import GeometricMedian
+from openfl.component.aggregation_functions import WeightedAverage
+from openfl.component.assigner.custom_assigner import Assigner
+from openfl.component.assigner.tasks import TrainTask
+from openfl.component.assigner.tasks import ValidateTask
+
+
+def assigner_function(collaborators, round_number, **kwargs):
+    """Return tasks by collaborator."""
+    tasks_by_collaborator = {}
+    for collaborator in collaborators:
```

Comment:
could we also introduce tests for the case when the collaborator `two` is assigned just the validation task?

Reply:
I thought about it, but looks like there're no difference for assigner and that tests will check only test assigner function, not assigner.